### PR TITLE
Add DiscordBots.org command

### PR DESCRIPTION
**Pull request by QuantumlyTangled**
Add command which allows getting a bots header image.

### CODE REVIEW

File: commands/Tools/dbl.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+			cooldown: 15,
+			usage: '<target:user>',
+			usageDelim: ''
+		});
+	}
+
+	async run(msg, [target]) {
+		let url = '';
+		if (!target.bot) throw 'You need to provide a bot id and not a user one silly';
+		const type = msg.flags.type ? String(msg.flags.type) : 'large';
```
Comment (kyranet) [inline commands/Tools/dbl.js]: `msg.flags.type || 'large'`

File: commands/Tools/dbl.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+			case 'owner': url = `https://discordbots.org/api/widget/owner/${target.id}.png${icon}`;
+				break;
+			default: url = `https://discordbots.org/api/widget/${target.id}.png`;
+		}
+		const image = await fetch(url)
+			.then(response => response.buffer())
+			.catch(() => {
+				throw 'I could not download the file. Can you try again with another image?';
+			});
+		return msg.send(new MessageAttachment(image, `${target.id}.png`));
```
Comment (kyranet) [inline commands/Tools/dbl.js]: msg.channel.sendFile, and remove MessageAttachment requirement

File: commands/Tools/dbl.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+			usageDelim: ''
+		});
+	}
+
+	async run(msg, [target]) {
+		let url = '';
+		if (!target.bot) throw 'You need to provide a bot id and not a user one silly';
+		const type = msg.flags.type ? String(msg.flags.type) : 'large';
+		const icon = msg.flags.icon ? '?noavatar=true' : '';
+		switch (type) {
```
Comment (kyranet) [inline commands/Tools/dbl.js]: Can refactored to ``https://discordbots.org/api/widget${type ? `/${type}` : ''}/${target.id}.png${icon}``, and make type default to null/empty string

**Comment by kyranet** [comment]
Bump, requested changes a long time ago.

**Comment by QuantumlyTangled** [comment]
Oh yes from the times of hacktoberfest

**Comment by kyranet** [comment]
Closing due to the age of the requested changes, feel free to make a new PR later.

**Pull request closed without merging**